apps/module-scan/src/interpreter.ts

```diff
@@ -373,10 +404,6 @@ export default class Interpreter {
     }
   }
 
-  public electionDidChange(): void {
-    this.hmpbInterpreter = undefined
-  }
-
```

Comment:
just being paranoid, why did we take this out?

Reply:
Unused, wanted to be clear that the hash couldn't get invalidated.